app/src/main/java/org/astraea/metrics/kafka/metrics/modifiers/HasCount.java

```diff
@@ -0,0 +1,9 @@
+package org.astraea.metrics.kafka.metrics.modifiers;
+
+import java.util.Objects;
+
+public interface HasCount extends HasBeanObject {
+  default long count() {
+    return (long) Objects.requireNonNull(beanObject().getAttributes().get("Count"));
```

Comment:
`null`是合法的值，因此這樣會造成不該發生的NPE

Reply:
> null是合法的值

不好意思，學長的意思是指 Count 的形態  ``Long`` 的值域內 null 是合法的，但是轉型 ``(long)`` 發生時會自動產生一個類似下面的 Null Pointer Exception，所以 ``Objects.requireNonNull`` 是多餘的嗎

```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.Long.longValue()" because "null" is null
```

Reply:
我指的是count對標的值可能不存在，這在topic剛建立的時候是可能發生的。因此從attribute拿值的時候可能找不到count，遇到此狀況應該要回傳0而非觸發NPe